ui/src/tool-panel/km-post/dialog/km-post-edit-dialog.tsx

```diff
@@ -343,10 +329,18 @@ export const KmPostEditDialog: React.FC<KmPostDialogProps> = (props: KmPostDialo
             {props.kmPostId && draftDeleteConfirmationVisible && (
                 <KmPostDeleteConfirmationDialog
                     id={props.kmPostId}
-                    onCancel={closeDraftDeleteConfirmation}
-                    onClose={onKmPostDeleted}
+                    onClose={() => setDraftDeleteConfirmationVisible(false)}
+                    onSave={props.onSave}
                 />
             )}
         </React.Fragment>
     );
 };
+
+function findTrackNumberKmPost(state: KmPostEditState): Promise<LayoutKmPost | undefined> {
+    if (state.kmPost?.trackNumberId !== undefined && isValidKmNumber(state.kmPost.kmNumber)) {
+        return getKmPostByNumber('DRAFT', state.kmPost.trackNumberId, state.kmPost.kmNumber, true);
+    } else {
+        return Promise.resolve(undefined);
+    }
+}
```

Comment:
Ei nyt suoraan tämän tiketin asiaa koska tämä taitaa olla pelkkä siirto, mutta jäin miettimään että kannattaisiko nykymallin sijasta vaan hakea kaikki ratanumeron kilometripylväät (changeTime niille depsuksi) ja sen jälkeen tehdä tämä tarkastelu ihan kokonaan vaan frontin päässä 🤔 Ratanumeron kilometripylväiden haku on kuitenkin aika kevyt operaatio bäkkärin osalta, niin ei tarvisi niin miljoonaa bäkkärikutsua (plus tätä käytetään kuitenkin validoinneissa -> validointivirhe voidaan kumminkin näyttää nykyisellään vasta kun bäkkärikutsu palaa, ja tässä voi jossain erikoistilanteissa kestää pitkään)

Reply:
Joo tämä on tosiaan vain siirto, mutta tuo olisi kyllä ihan vaihtoehto. Ratanumeron pylväät voi myös kakuttaa frontissa, jolloin haut jää harvoiksi.